services/cd-service/pkg/repository/repository.go

```diff
@@ -725,27 +725,34 @@ func (r *repository) ProcessQueueOnce(ctx context.Context, e transformerBatch, c
 	// Try pushing once
 	err = r.Push(e.ctx, pushAction(pushOptions, r))
 	if err != nil {
+		logger.Sugar().Infof("applyTransformerBatches: push %d error: %v", len(transformerBatches), err)
 		gerr, ok := err.(*git.GitError)
 		// If it doesn't work because the branch diverged, try reset and apply again.
 		if ok && gerr.Code == git.ErrorCodeNonFastForward {
 			err = r.FetchAndReset(e.ctx)
+			logger.Sugar().Infof("applyTransformerBatches: push %d with error C: %v", len(transformerBatches), err)
```

Comment:
Are these logs needed or were they just for debugging?

Reply:
I removed them